api/src/main/java/org/motechproject/nms/api/web/contract/FlwUserResponse.java

```diff
@@ -68,6 +68,17 @@ public void setMaxAllowedEndOfUsagePrompt(int maxAllowedEndOfUsagePrompt) {
     }
 
     @Override
+    public String toString() {
+        return "FlwUserResponse{" +
+                "currentUsageInPulses=" + currentUsageInPulses +
+                ", endOfUsagePromptCounter=" + endOfUsagePromptCounter +
+                ", welcomePromptFlag=" + welcomePromptFlag +
+                ", maxAllowedUsageInPulses=" + maxAllowedUsageInPulses +
+                ", maxAllowedEndOfUsagePrompt=" + maxAllowedEndOfUsagePrompt +
+                "} " + super.toString();
```

Comment:
intellij has that as one of the options when you tell it to insert
toString()

On Thu, Aug 6, 2015 at 8:14 AM, Frank Huster <notifications@github.com>
wrote:

> In
> api/src/main/java/org/motechproject/nms/api/web/contract/FlwUserResponse.java
> <https://github.com/motech-implementations/mim/pull/557#discussion_r36426154>
> :
>
> > @@ -68,6 +68,17 @@ public void setMaxAllowedEndOfUsagePrompt(int maxAllowedEndOfUsagePrompt) {
> >      }
> >
> >      @Override
> > +    public String toString() {
> > +        return "FlwUserResponse{" +
> > +                "currentUsageInPulses=" + currentUsageInPulses +
> > +                ", endOfUsagePromptCounter=" + endOfUsagePromptCounter +
> > +                ", welcomePromptFlag=" + welcomePromptFlag +
> > +                ", maxAllowedUsageInPulses=" + maxAllowedUsageInPulses +
> > +                ", maxAllowedEndOfUsagePrompt=" + maxAllowedEndOfUsagePrompt +
> > +                "} " + super.toString();
>
> I never saw a super.toString inside a toString before, cool. and a bit
> weird.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/557/files#r36426154>.
>